Makefile

```diff
@@ -78,6 +79,15 @@ $(warning No code-sign identity found (CODE_SIGN_IDENTITY is not set))
 endif
 endif
 
+ifeq ($(TARGET_PLATFORM),win32)
+ifndef CODE_SIGN_CERTIFICATE
+$(warning No code-sign certificate found (CODE_SIGN_CERTIFICATE is not set))
+endif
+ifndef CODE_SIGN_CERTIFICATE_PASSWORD
+$(warning No code-sign certificate password found (CODE_SIGN_CERTIFICATE_PASSWORD is not set))
+endif
```

Comment:
Probably makes sense to only show these warnings when you're actually about to do the code-signing step, rather than showing them all the time to all users.